src/ValueParsers/YearMonthTimeParser.php

```diff
@@ -58,34 +64,56 @@ public function __construct(
 	 * @return TimeValue
 	 */
 	protected function stringParse( $value ) {
+		$trimmedValue = trim( $value );
+		switch ( $trimmedValue[0] ) {
```

Comment:
This will issue a PHP Notice if `$trimmedValue` is the empty string. I would suggest either `$trimmedValue[0] ?? ''` or `substr( $trimmedValue, 0, 1 )`. And ideally add a test for this, too, so that no one in the future replaces the `substr( $trimmedValue, 0, 1 )` with `$trimmedValue[0]` again because it’s obviously equivalent ;)

Reply:
Did it. I would like to avoid doing this in the first place and have the EraParser only handle the "CE/BCE" stuff but at the moment there is no other way.